src/main/scala/no/ndla/articleapi/controller/ArticleController.scala

```diff
@@ -86,8 +120,34 @@ trait ArticleController {
         case Some(image) => image
         case None => halt(status = 404, body = Error(Error.NOT_FOUND, s"No article with id $articleId found"))
       }
+    }
 
+    post("/", operation(newArticle)) {
+      val newArticle = extract[NewArticle](request.body)
+      val article = updateService.newArticle(newArticle)
+      logger.info(s"CREATED article with ID = ${article.id}")
+      Created(body=article)
+    }
+
+    patch("/:article_id", operation(updateArticle)) {
+      val articleId = long("article_id")
+      val updatedArticle = extract[UpdatedArticle](request.body)
+      val article = updateService.updateArticle(articleId, updatedArticle)
```

Comment:
Siden `updateService` returnerer en `None` dersom det ikke finnes noen artikkel med gitt id, så må vi her håndtere en `None` og returnere en `404 Not Found`